decidim-core/app/cells/decidim/upload_modal_cell.rb

```diff
@@ -98,9 +98,15 @@ def input_validation_field
     end
 
     def explanation
-      return I18n.t("explanation", scope: options[:help_i18n_scope], attribute:) if options[:help_i18n_scope].present?
+      return I18n.t("explanation", scope: options[:help_i18n_scope], attribute: attribute_translation) if options[:help_i18n_scope].present?
 
-      I18n.t("explanation", scope: "decidim.forms.upload_help", attribute:)
+      I18n.t("explanation", scope: "decidim.forms.upload_help", attribute: attribute_translation)
+    end
+
+    def attribute_translation
+      I18n.t(attribute, scope: [:activemodel, :attributes, resource_class.constantize.model_name.param_key].join("."))
+    rescue NameError
+      I18n.t(attribute, scope: "decidim.forms")
```

Comment:
In this rescue section maybe we should return something safer as I don't know if the translation exists in this scope

Reply:
I would leave this as it is to make sure that any new fields that is being added will force the developer to add a description. Maybe I would change the scope to be "activemodel.attributes" instead of "decidim.forms"